custom_components/ramses_cc/water_heater.py

```diff
@@ -82,7 +85,7 @@ def entity_factory(entity_class, broker, device):  # TODO: deprecate
             platform.async_register_entity_service(name, schema, f"svc_{name}")
 
 
-class EvohomeDHW(EvohomeZoneBase, WaterHeaterEntity):
+class RamsesWaterHeater(EvohomeZoneBase, WaterHeaterEntity):
```

Comment:
May as well call this **RamsesZoneBase** too.

Reply:
Yep. I was going to come back to a few other renames once #94 is merged.

Reply:
ok, lets merge this for now